sdk/clientcore/core/src/main/java/io/clientcore/core/models/binarydata/BinaryData.java

```diff
@@ -235,63 +234,66 @@ public static BinaryData fromBytes(byte[] data) {
-     * changes to a {@link ByteBuffer} outside of this instance will result in the contents of this BinaryData instance
-     * being updated as well. To safely update the byte array without impacting the BinaryData instance, perform an
-     * array copy first.</p>
-     *
-     * <p><strong>Create an instance from a List&lt;ByteBuffer&gt;</strong></p>
-     *
-     * <!-- src_embed io.clientcore.core.util.BinaryData.fromListByteBuffer#List -->
-     * <pre>
-     * final List&lt;ByteBuffer&gt; data = Stream.of&#40;&quot;Some &quot;, &quot;data&quot;&#41;
-     *     .map&#40;s -&gt; ByteBuffer.wrap&#40;s.getBytes&#40;StandardCharsets.UTF_8&#41;&#41;&#41;
-     *     .collect&#40;Collectors.toList&#40;&#41;&#41;;
-     * BinaryData binaryData = BinaryData.fromListByteBuffer&#40;data&#41;;
-     * System.out.println&#40;binaryData&#41;;
-     * </pre>
-     * <!-- end io.clientcore.core.util.BinaryData.fromListByteBuffer#List -->
-     *
-     * @param data The {@link List} of {@link ByteBuffer} that {@link BinaryData} will represent.
-     * @return A {@link BinaryData} representing the {@link List} of {@link ByteBuffer}.
-     */
-    public static BinaryData fromListByteBuffer(List<ByteBuffer> data) {
-        return new ListByteBufferBinaryData(data);
-    }
+    //    /**
+    //     * Creates an instance of {@link BinaryData} from the given {@link List} of {@link ByteBuffer}.
+    //     *
+    //     * <p>The input {@link ByteBuffer} instances are used as a reference by this instance of {@link BinaryData} and any
+    //     * changes to a {@link ByteBuffer} outside of this instance will result in the contents of this BinaryData instance
+    //     * being updated as well. To safely update the byte array without impacting the BinaryData instance, perform an
+    //     * array copy first.</p>
+    //     *
+    //     * <p><strong>Create an instance from a List&lt;ByteBuffer&gt;</strong></p>
+    //     *
+    //     * <!-- src_embed io.clientcore.core.util.BinaryData.fromListByteBuffer#List -->
+    //     * <pre>
+    //     * final List&lt;ByteBuffer&gt; data = Stream.of&#40;&quot;Some &quot;, &quot;data&quot;&#41;
+    //     *     .map&#40;s -&gt; ByteBuffer.wrap&#40;s.getBytes&#40;StandardCharsets.UTF_8&#41;&#41;&#41;
+    //     *     .collect&#40;Collectors.toList&#40;&#41;&#41;;
+    //     * BinaryData binaryData = BinaryData.fromListByteBuffer&#40;data&#41;;
+    //     * System.out.println&#40;binaryData&#41;;
+    //     * </pre>
+    //     * <!-- end io.clientcore.core.util.BinaryData.fromListByteBuffer#List -->
+    //     *
+    //     * @param data The {@link List} of {@link ByteBuffer} that {@link BinaryData} will represent.
+    //     * @return A {@link BinaryData} representing the {@link List} of {@link ByteBuffer}.
+    //     */
+    //    public static BinaryData fromListByteBuffer(List<ByteBuffer> data) {
```

Comment:
Can we just make this package-private instead of commenting it out?

Reply:
Sure